prov/efa/src/efa_fabric.c

```diff
@@ -136,17 +136,6 @@ int efa_fabric(struct fi_fabric_attr *attr, struct fid_fabric **fabric_fid,
 	if (ret)
 		goto err_free_fabric;
 
-	/* Open shm provider's fabric domain */
```

Comment:
Is there a reason to open up the shm resources at domain open instead of fabric open?